proto/dsmr/dsmr.proto

```diff
@@ -16,14 +16,6 @@ message GetChunkResponse {
   bytes chunk = 1;
 }
 
-message GetChunkSignatureRequest {
-  bytes chunk = 1;
-}
-
-message GetChunkSignatureResponse {
-  bytes signature = 1;
-}
-
```

Comment:
I don't know why these were added to begin with since we use acp118 signature requests